omero/sysadmins/unix/server-install-homebrew.txt

```diff
@@ -157,158 +185,66 @@ able to build the server, i.e.
--------------
-
-Database creation
-^^^^^^^^^^^^^^^^^
-
-#. Create a new database cluster::
-
-    $ initdb -E UTF8 /usr/local/var/postgres
-
-#. Start the PostgreSQL server.
-
-   If you wish to use ``brew services`` to start postgresql, you will
-   first need to install it, then start the service::
-
-    $ brew tap gapple/services
-    $ brew services start postgresql
-
-   Alternatively, to start by hand::
-
-    $ pg_ctl -D /usr/local/var/postgres/ -l /usr/local/var/postgres/server.log start
-
-   It is also possible to configure :program:`launchd` to start
-   PostgreSQL at boot
-
-#. Create a user and database::
-
-    $ createuser -P -D -R -S db_user
-    Enter password for new role:       # enter db_password
-    Enter it again:                    # enter db_password
-    $ createdb -E UTF8 -O db_user omero_database
+    $ cd ~/code/projects/OMERO/dist
 
-   Note ``db_user`` and ``db_password`` should be replaced with a
-   username and password of your choice.  Record these for future use.
+and follow the steps for setting up the database and OMERO data directory as mentioned
+in the previous section.
 
-#. Check to make sure the database has been created.
-
-::
-
-    $ psql -h localhost -U db_user -l
-
-This command should give similar output to the following::
-
-                            List of databases
-
-       Name         | Owner   | Encoding |  Collation  |    Ctype    | Access privileges
-    ----------------+---------+----------+-------------+-------------+-------------------
-     omero_database | db_user | UTF8     | en_GB.UTF-8 | en_GB.UTF-8 |
-     postgres       | ome     | UTF8     | en_GB.UTF-8 | en_GB.UTF-8 |
-     template0      | ome     | UTF8     | en_GB.UTF-8 | en_GB.UTF-8 | =c/ome           +
-                    |         |          |             |             | ome=CTc/ome
-     template1      | ome     | UTF8     | en_GB.UTF-8 | en_GB.UTF-8 | =c/ome           +
-                    |         |          |             |             | ome=CTc/ome
-    (4 rows)
-
-OMERO.server
-^^^^^^^^^^^^
-
-Now configure OMERO.server to connect to the newly-created database::
-
-    $ omero config set omero.db.name omero_database
-    $ omero config set omero.db.user db_user
-    $ omero config set omero.db.pass db_password
-
-And then, generate the database schema::
-
-    $ omero db script --password secretpassword
+OMERO.web
+^^^^^^^^^
 
-You should see output similar to this:
+Basic setup for OMERO using nginx::
 
-.. literalinclude:: /downloads/cli/db-script-example.txt
+    $ export HTTPPORT=${HTTPPORT:-8080}
+    $ omero web config nginx-development --http $HTTPPORT > $(brew --prefix omero52)/etc/nginx.conf
 
-Then run the SQL commands in the generated schema file to create the
-database:
+:download:`step03_nginx.sh <walkthrough/osx/step03_nginx.sh>`
 
-.. parsed-literal::
+For detailed instructions on how to deploy OMERO.web in a production environment such as Apache or
+Nginx please see :doc:`install-web`.
```

Comment:
This sentence belongs above the ``Basic setup for OMERO using nginx:``.  I understand that you have to install nginx first, only then to run the two lines in the ``Basic setup....`` or the script03.